### PR TITLE
feat(layouts): add boilerplate layouts

### PR DESCRIPTION
**Pull request by CITguy**
* Vertical Layout (using holy grail flexbox)
* Horizontal Layout (using CSS Grid)
* Update Install guide to include information about polyfill.io
* Update CSS selectors in tests

JIRA: https://jira.rax.io/browse/SURF-623

### LGTM's
- [x] Dev LGTM
- [x] Design LGTM
- [x] Zoom LGTM

### Screenshots
_Updated Installation Guide_
localhost_3000_helix-ui_guides_install_

_Components > Layouts_
localhost_3000_helix-ui_components_layouts_

_Components > Layouts > Vertical_
localhost_3000_helix-ui_components_layouts_vertical-layout-template html

_Components > Layouts > Horizontal_
localhost_3000_helix-ui_components_layouts_horizontal-layout-template html

_Components > Layouts > Horizontal (height constrained; scrolling nav)_
localhost_3000_helix-ui_components_layouts_horizontal-layout-template html-scrollnav

_Side Rails Before &amp; After_

Before | After
----- | -----
rackerlabs github io_helix-ui_guides_install_ screenshot | localhost_3000_helix-ui_guides_install_after

### CODE REVIEW

File: docs/_templates/partials/footer_nav.njk [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="hxCol">
-  &copy; 2017 Rackspace US, Inc.
+  &copy; 2018 Rackspace US, Inc.
```
Comment (CITguy) [inline docs/_templates/partials/footer_nav.njk]: Might as well update this while I'm in the neighborhood.

File: docs/docs.less [hunk 2]
```diff
@@ -14,8 +14,16 @@
 */
 @import (reference) 'vars';
 
-body {
-  background-color: @gray-25;
```
Comment (CITguy) [inline docs/docs.less]: moved to `src/helix-ui/styles/components/layouts.less`

File: docs/docs.less [hunk 3]
```diff
@@ -14,8 +14,16 @@
...
+.brand-logo {
```
Comment (CITguy) [inline docs/docs.less]: Pulled in from `src/helix-ui/styles/layouts.less` (older implementation of layouts)

File: docs/styles/demo/nav-docs.less [hunk 4]
```diff
@@ -0,0 +1,5 @@
+.nav-demo {
+  .hxNav {
+    width: 15rem;
```
Comment (CITguy) [inline docs/styles/demo/nav-docs.less]: `.hxNav` applies visual appearance, but `#nav` handles the width, so we imitate the width in the demo to maintain appearances in the layout.

File: src/helix-ui.less [hunk 5]
```diff
@@ -37,11 +37,14 @@ html {
 }
 
 body {
+  background-color: @gray-25;
```
Comment (CITguy) [inline src/helix-ui.less]: pulled from `docs/docs.less`

File: src/helix-ui.less [hunk 6]
```diff
@@ -37,11 +37,14 @@ html {
...
   color: @gray-900;
   font-family: "Roboto", "Helvetica Neue", Helvetica, Arial, sans-serif;
   font-size: 1rem;
   font-weight: 400;
+  height: 100%;
```
Comment (CITguy) [inline src/helix-ui.less]: pulled from older layout implementation

File: src/helix-ui/styles/layout.less [hunk 7]
```diff
@@ -1,60 +0,0 @@
-.hxApp {
```
Comment (CITguy) [inline src/helix-ui/styles/layout.less]: all `.hxApp` prefixed styles are gone

File: src/helix-ui/styles/layout.less [hunk 8]
```diff
@@ -1,60 +0,0 @@
...
-    width: @app-nav-width;
-  }
-
-  &__content {
-    flex-grow: 1;
-    padding: 1em;
-  }
-}
-
-.brand-logo {
```
Comment (CITguy) [inline src/helix-ui/styles/layout.less]: moved to `docs/docs.less`

File: src/helix-ui/styles/navigation.less [hunk 9]
```diff
@@ -14,13 +14,11 @@
   font-size: 0.875rem; /* ~14px */
   letter-spacing: 0.75px;
   padding-top: 2rem;
-  width: 15rem; /* ~240px */
```
Comment (CITguy) [inline src/helix-ui/styles/navigation.less]: layout styling handles the width, not the navigation component

File: src/helix-ui/styles/navigation.less [hunk 10]
```diff
@@ -14,13 +14,11 @@
...
 
   hx-disclosure {
     color: inherit;
     display: block;
     padding: @vpad @rpad @vpad @l1-indent;
-    text-transform: uppercase;
```
Comment (CITguy) [inline src/helix-ui/styles/navigation.less]: Refactored to `> hx-disclosure` to ensure it only applies to the top level items.

File: docs/_templates/guide.njk [hunk 11]
```diff
@@ -1,18 +1,14 @@
...
-        {% block content %}
-          {# page content goes here #}
-        {% endblock %}
-      </div>
-    </div>
-  </article>
+    {% block content %}
+      {# page content goes here #}
+    {% endblock %}
+  </main>
```
[the remark this answers is not in the document]
Comment (CITguy) [reply docs/_templates/guide.njk]: Apparently there's nothing to fix. just an illusion with the way github displays it

File: docs/_templates/styleguide.njk [hunk 12]
```diff
@@ -1,18 +1,14 @@
...
-        {% block content %}
-          {# page content goes here #}
-        {% endblock %}
-      </div>
-    </div>
-  </article>
+    {% block content %}
+      {# page content goes here #}
+    {% endblock %}
+  </main>
```
[the remark this answers is not in the document]
Comment (CITguy) [reply docs/_templates/styleguide.njk]: Apparently there's nothing to fix. just an illusion with the way github displays it

File: docs/guides/install/index.html [hunk 13]
```diff
@@ -78,40 +77,6 @@ <h2 class="hxSectionTitle" id="web-components">Web Components</h2>
     </p>
   </section>
 
-  <section><!-- TL;DR -->
```
Comment (CITguy) [inline docs/guides/install/index.html]: Removed section. It was repetitive and is essentially replaced with the new layout templates.

File: src/helix-ui/styles/components/layouts.less [hunk 14]
```diff
@@ -0,0 +1,145 @@
...
+
+#nav {
+  flex-shrink: 0;
+  width: @app-nav-width;
+}
+
+#sidebar {
+  background-color: @gray-100;
+  flex-shrink: 0;
+  width: 25%; // 3 of 12 columns
```
Comment (CITguy) [inline src/helix-ui/styles/components/layouts.less]: Updated per redline specifications.

**Comment by CITguy** [comment]
@davismartin: Updates have been made. Please review.

**Comment by technabors** [comment]
Ran repo locally and tested with @araiford. We have checked the Design LGTM box.

File: docs/guides/install/index.html [hunk 15]
```diff
@@ -236,4 +201,17 @@ <h2 class="hxSectionTitle" id="bootstrap-theme">Bootstrap Theme</h2>
     </tbody>
   </table>
 </section>
+
+<section><!-- Supporting Legacy Browsers -->
+  <h2 class="hxSectionTitle" id="maximum-compatibility">Maximum Compatibility</h2>
+  <p>
+    To ensure maximum functionality for legacy browsers (i.e. bring IE
```
Comment (nimbinatus) [inline docs/guides/install/index.html]: Nit: s/`i.e.`/`i.e.,`

**Pull request merged**